src/test/java/de/tum/in/www1/artemis/architecture/ResourceArchitectureTest.java

```diff
@@ -69,6 +73,32 @@ void shouldCorrectlyUseRequestMappingAnnotations() {
         }
     }
 
+    @Test
+    void testUseKebabCaseForRestEndpoints() {
```

Comment:
This test currently fails:
> de.tum.in.www1.artemis.web.rest.UserResource.getVcsAccessToken(java.lang.Long) violates rule to use kebab case for rest call annotations

Reply:
yep, the refactor for these is in https://github.com/ls1intum/Artemis/pull/9182. When it is merged it will fix this here. I removed the ready for review label until then

Reply:
That should be fixed now :+1: